src/composer/prover.rs

```diff
@@ -348,18 +349,35 @@ impl Prover {
 
         // split quotient polynomial into 4 degree `n` polynomials
         let domain_size = domain.size();
-        let t_low_poly = FftPolynomial::from_coefficients_vec(
-            t_poly[0..domain_size].to_vec(),
-        );
-        let t_mid_poly = FftPolynomial::from_coefficients_vec(
-            t_poly[domain_size..2 * domain_size].to_vec(),
-        );
-        let t_high_poly = FftPolynomial::from_coefficients_vec(
-            t_poly[2 * domain_size..3 * domain_size].to_vec(),
-        );
-        let t_4_poly = FftPolynomial::from_coefficients_vec(
-            t_poly[3 * domain_size..].to_vec(),
-        );
+
+        let mut t_low_vec = t_poly[0..domain_size].to_vec();
+        let mut t_mid_vec = t_poly[domain_size..2 * domain_size].to_vec();
+        let mut t_high_vec = t_poly[2 * domain_size..3 * domain_size].to_vec();
+        let mut t_4_vec = t_poly[3 * domain_size..].to_vec();
+
+        // select 3 blinding factors for the quotient splitted polynomials
+        let b_10 = BlsScalar::random(&mut *rng);
+        let b_11 = BlsScalar::random(&mut *rng);
+        let b_12 = BlsScalar::random(&mut *rng);
+
+        // t_low'(X) + b_10*X^n
+        t_low_vec.push(b_10);
+
+        // t_mid'(X) - b_10 + b_11*X^n
+        t_mid_vec[0] = t_mid_vec[0] - b_10;
```

Comment:
`cargo clippy` will complain with this line. Try:
```suggestion
        t_mid_vec[0] -= b_10;
```

Reply:
it doesn't! But done! :)